packages/twenty-front/src/utils/validation-schemas/safeStringArrayJSONSchema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+export const safeStringArrayJSONSchema = z
```

Comment:
Ok to extract this as it was but the validation seems to introduce a silent error that returns [], could we throw or  return a wrong validation state instead when the value is really not parsable as an array ?